app-creator/client/src/utils/helpers.ts

```diff
@@ -258,25 +272,41 @@ app.draw();
  * Normalize template.
  */
 export function normalizeTemplate(template: AppCreatorStore['template']) {
-  const {widgets} = template;
+  const { widgets } = template;
   for (const id in widgets) {
     const type = getWidgetType(id);
     if (type === WidgetType.SELECT) {
       const items = widgets[id].uniqueAttributes.items;
-      widgets[id].uniqueAttributes.items =
-          getArrayStringFromCommaSeparatedValues(items);
+      widgets[
+        id
+      ].uniqueAttributes.items = getArrayStringFromCommaSeparatedValues(items);
     } else if (type === WidgetType.CHART) {
       const colors = widgets[id].uniqueAttributes.color;
-      widgets[id].uniqueAttributes.color =
-          getArrayStringFromCommaSeparatedValues(colors);
+      widgets[
+        id
+      ].uniqueAttributes.color = getArrayStringFromCommaSeparatedValues(colors);
       const dataTable = widgets[id].uniqueAttributes.dataTable;
       widgets[id].uniqueAttributes.dataTable = normalizeDataTable(dataTable);
+    } else if (type === WidgetType.MAP) {
+      const customMapStyles = widgets[id].uniqueAttributes.customMapStyles;
+      if (customMapStyles !== '') {
+        widgets[id].uniqueAttributes.customMapStyles = normalizeCustomMapStyles(
+          customMapStyles
+        );
+      }
     }
   }
 
   return template;
 }
 
+/**
+ * Removes white spaces and adds the necessary escape characters for parsing.
+ */
+function normalizeCustomMapStyles(customMapStyles: string) {
+  return customMapStyles.replace(/[\t\n\r ]/g, '').replace(/\"/g, '\\"');
```

Comment:
`\s` matches these characters and more: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Character_Classes